lib/index.js

```diff
@@ -0,0 +1,18 @@
+const helmet = require('helmet');
+const ONE_YEAR = 31536000000;
+
+module.exports = () => httpsOnlyMiddleware;
+
+function httpsOnlyMiddleware(req, res, next) {
+	if (req.headers['x-forwarded-proto'] === 'http') {
+		res.redirect(301, 'https://' + req.headers.host + req.url);
+		return;
+	}
+	helmet.hsts({
+		maxAge: ONE_YEAR,
+		includeSubDomains: true,
```

Comment:
Fixed a typo/breaking change [here](https://github.com/debitoor/httpsonly/pull/1/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L11) as it was previously spelled like `includeSubdomains`.

Reply:
I think this is not "breaking change"
before: https://github.com/helmetjs/hsts/blob/v1.0.0/index.js#L37-L39
after: https://github.com/helmetjs/helmet/blob/main/middlewares/strict-transport-security/index.ts#L31-L35